doc/manpages/dinit-service.5.m4

```diff
@@ -360,6 +360,23 @@ This setting has no effect if the service is set to run on the console (via the
 \fBstarts\-on\-console\fR, or \fBshares\-console\fR options).
 The value is subject to variable substitution (see \fBVARIABLE SUBSTITUTION\fR).
 .TP
+\fBlogfile\-permissions\fR = \fIoctal-permissions-mask\fR
+Gives the permissions for the logfile specified using \fBlogfile\fR. Normally this will be 600 (user access
```

Comment:
These values don't seem normal for a logfile. Wouldn't it be more likely that read-only permission is granted to the group and/or all users?

Reply:
For example:
"Normally this will be 600 (user access only), 640 (also readable by the group), or 644 (readable by all users)."